docs/src/main/sphinx/release/release-466.md

```diff
@@ -0,0 +1,112 @@
+# Release 466 (27 Nov 2024)
+
+## General
+
+* Improve performance of queries with grouping on joins. ({issue}`23812`)
+* TBD Spooling protocol is stable. Link to docs! ({issue}`24214`)
+* TBD Improve performance of querying `system.jdbc.tables` for multiple catalogs. ({issue}`24159`)
+* Support partial update. ({issue}`24075`)
+
+## Security
+
+* Add support [data access control with Apache
+  Ranger](/security/apache-ranger-access-control), including support for
+  column-masking, row-filtering, and audit logging. ({issue}`22675`)
+
+## Web UI
+
+## JDBC driver
+
+## Server RPM
+
+## Docker image
+
+## CLI
+
+## BigQuery connector
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+## Druid connector
+
+## Elasticsearch connector
+
+## Exasol connector
+
+## Faker connector
+
+## Google Sheets connector
+
+## Hive connector
+
+* TBD Improve performance of querying `system.jdbc.tables` for Hive, Iceberg, and Delta. ({issue}`24110`)
```

Comment:
Asked in original PR for clarification so I can reword .. also if it applies to Hudi

Reply:
Not to hudi.